rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf

```diff
@@ -1065,6 +1069,47 @@ SecRule REQUEST_HEADERS:Referer "@rx ^[^\.]*?(?:['\*\?\x5c`][^\n/]+/|/[^/]+?['\*
             setvar:'tx.rce_score=+%{tx.critical_anomaly_score}',\
             setvar:'tx.inbound_anomaly_score_pl2=+%{tx.critical_anomaly_score}'"
 
+#
+# -=[ Rule 932207 ]=-
+#
+# Sibling of 932200 targeting the User-Agent header. URLs cause false positives in rule 932200
+# and must be handled with additional checks.
+# This rule targets either the query string of a URL or a URL followed by a semi-colon,
+# e.g., `www.google.com;c$$at+/et$$c/pas$$swd`.
+#
+# Regular expression generated from regex-assembly/932207.ra.
+# To update the regular expression run the following shell script
+# (consult https://coreruleset.org/docs/development/regex_assembly/ for details):
+#   crs-toolchain regex update 932207
+#
+SecRule REQUEST_HEADERS:User-Agent "@rx ^[^\.]+\.[^;\?]+[;\?](.*(['\*\?\x5c`][^\n/]+/|/[^/]+?['\*\?\x5c`]|\$[!#-\$\(\*\-0-9\?-\[_a-\{]))" \
+    "id:932207,\
+    phase:2,\
+    block,\
+    capture,\
+    t:none,t:lowercase,t:urlDecodeUni,\
+    msg:'RCE Bypass Technique',\
+    logdata:'Matched Data: %{TX.2} found within %{TX.932205_MATCHED_VAR_NAME}: %{MATCHED_VAR}',\
+    tag:'application-multi',\
+    tag:'language-multi',\
+    tag:'platform-multi',\
+    tag:'attack-rce',\
+    tag:'paranoia-level/2',\
+    tag:'OWASP_CRS',\
+    tag:'capec/1000/152/248/88',\
+    tag:'PCI/6.5.2',\
+    ver:'OWASP_CRS/4.0.0-rc2',\
+    severity:'CRITICAL',\
+    setvar:'tx.932205_matched_var_name=%{matched_var_name}',\
```

Comment:
Variable sould be named tx.932207_...

Reply:
BTW, we probably don't need this here - there's only one variable so we can hardcode `REQUEST_HEADERS:User-Agent`.